packages/errors/src/errors.ts

```diff
@@ -1330,7 +1330,26 @@ export const AllCypressErrors = {
     `
   },
 
-  TEST_FILES_DEPRECATION: (errShape: BreakingErrResult) => {
+  MIGRATED_CONFIG_OPTIONS_SPEC_PATTERN: ({ name, setupNodeEvents = false }: {name: string, setupNodeEvents?: boolean}, err?: Error) => {
```

Comment:
I think you've missed a few fields.

- `ignoreTestFiles` -> `ignoreSpecPattern`

Also, how about things like "supportFile and baseUrl are not valid root values". Do we want to handle those, too?